runbook.md

```diff
@@ -0,0 +1,17 @@
+# Runbook for proposed changes with steps
+
+## -- changes:
+ 1. secure access to prod trust claim DB
+ 2. change digital ocean droplet to test server with a separate DB for trust claim
+ 3. maintain digital ocean for Taiga and Aws for trust claim
+ 4. set-up CI/CD on dev (digital ocean trustclaim server)
+ 5. rewrite taiga.whatscookin.us to redirect to Taiga.linkedtrust.us
+    
+## -- steps:
+ 1. for secure access to prod DB, I'll only allow the Aws prod IP to be the only inbound traffic IP
+2. for this, I'll enable the DNS name 'lives.linkedtrust.us' to be the dev domain and the IP being the dev IP, which is on CI/CD with the master branch on both both front and backend repo, also I'll need the digital ocean webhook to be attached to both front and backend repo for me to enable it with Jenkins.
```

Comment:
thi sis good, but for clarify lets change the dns to `dev.linkedtrust.us`  so its very clear which site we are on

a couple reasons - just general good practice, but also devs and testers need to know  - they can enter somewhat junky data on dev, never on live

otherwise, this is fine!

Reply:
this is the dev server, " https://dev.linkedtrust.us/ " then for the users, i wanted to ask if users created are stored in DB or where? but this dev server is connected to it's on server, and the prod db is secured too

Reply:
also for the ci cd, please help me to set up this webhook to only listen to the dev branch for just two events, push and merge to it " http//68.183.144.184/github-webhook/ "